iml-api/src/task.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{command::get_command, error::ImlApiError};
+use futures::TryFutureExt;
+use iml_orm::{task::ChromaCoreTask, tokio_diesel::AsyncRunQueryDsl as _};
+use iml_rabbit::Connection;
+use iml_wire_types::{ApiList, CmdWrapper};
+use warp::Filter;
+
+async fn create_task(
+    client: Connection,
+    task: serde_json::Value,
+) -> Result<impl warp::Reply, warp::Rejection> {
+    let pool = iml_orm::pool().map_err(ImlApiError::ImlR2D2Error)?;
```

Comment:
Doing this per call defeats the purpose of using a pool. 

We should instead use a warp filter to wrap this and pass it into each call (similar to the client filter).